src/services/search/currentMakeModels.ts

```diff
@@ -19,3 +19,15 @@ export const fetchCurrentModels = async ({
     options,
   })
 }
+
+export const fetchMakesSuggestions = async ({
+  query,
+  options = {},
+}: {
+  query: string
+  options?: ApiCallOptions
+}): Promise<MakeWithModels[]> =>
+  await fetchPath({
```

Comment:
I guess you don't wanna await that rather return it?

Reply:
both works, will change it. Since the function is declared as async, you have to await it anyway

Reply:
Oh, I missed that you used a shorthand arrow function on multiple lines...